docroot/modules/custom/foia_webform/tests/src/Kernel/FoiaSubmissionQueueHandlerTest.php

```diff
@@ -189,12 +190,11 @@ public function testFoiaRequestAttachmentPendingScan() {
     $webformSubmission->save();
     $this->webformSubmission = $webformSubmission;
 
-    $queuedSubmission = $this->foiaSubmissionsQueue->claimItem()->data;
+    $queueCount = $this->foiaSubmissionsQueue->numberOfItems();
 
-    $this->assertNotEmpty($queuedSubmission, "Expected a FOIA request ID to be queued, but nothing was found in the queue.");
-    $this->assertEquals('1', $queuedSubmission->id, "Queued FOIA Request ID does not match expected.");
+    $this->assertEquals($originalQueueCount, $queueCount, "Expect that the submission with attachment would not be queued, it was.");
 
-    $foiaRequest = FoiaRequest::load($queuedSubmission->id);
+    $foiaRequest = FoiaRequest::load($this->webformSubmission->id());
```

Comment:
I think this is working because the id of the webform submission happens to be the same as the id of the foia request. In reality though, the FOIA request will not have the same ID as the webform submission. Instead, we could either load the request by writing a query to find it based on the submission ID. Or we could use Drupal's entity storage and do a `loadByProperties` to load the foia request based on the webform submission ID.